ethereum/core/src/main/java/org/hyperledger/besu/ethereum/vm/StandardJsonTracer.java

```diff
@@ -128,14 +128,13 @@ public void traceExecution(
 
   private Gas computeCallGas(
       final AbstractCallOperation callOperation, final MessageFrame messageFrame) {
-    final GasCalculator gasCalculator = callOperation.gasCalculator();
     // as part of EIP 150 the returned costGas is gas - base * 63 / 64.
```

Comment:
non-blocking feedback, we should change this comment to be "cost" rather than "base".  AFAICT call cost includes memory expansion, value transfer, and new account creation costs